ios/brave-ios/Sources/Brave/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -427,4 +427,25 @@ public class AutocompleteTextField: UITextField, UITextFieldDelegate {
     applyCompletion()
     super.touchesBegan(touches, with: event)
   }
+
+  public override func pressesEnded(_ presses: Set<UIPress>, with event: UIPressesEvent?) {
+    guard isSelectionActive else {
+      super.pressesEnded(presses, with: event)
+      return
+    }
+
+    for press in presses {
+      guard let key = press.key else { continue }
+      let chars = key.charactersIgnoringModifiers
+      let isRightArrowChar = chars == UIKeyCommand.inputRightArrow
+      let isRightArrowKeyCode = key.keyCode == .keyboardRightArrow
+      if isRightArrowChar || isRightArrowKeyCode {
+        applyCompletion()
+        selectedTextRange = textRange(from: endOfDocument, to: endOfDocument)
```

Comment:
this shouldn't be needed since it happens in `applyCompletion()` (`removeCompletion()` returns true when `isSelectionActive` which is guarded against above).

Reply:
You're right! Removed.